clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -1690,10 +1690,17 @@ void SYCLIntegrationHeader::emitFwdDecl(raw_ostream &O, const Decl *D,
   PrintingPolicy P(D->getASTContext().getLangOpts());
   P.adjustForCPlusPlusFwdDecl();
   P.SuppressTypedefs = true;
+  P.SuppressUnwrittenScope = true;
   std::string S;
   llvm::raw_string_ostream SO(S);
   D->print(SO, P);
-  O << SO.str() << ";\n";
+  O << SO.str();
+
+  if (const auto *ED = dyn_cast<EnumDecl>(D)) {
+    QualType T = ED->getIntegerType();
+    O << " : " << T.getAsString() << ";\n";
+  } else
+    O << ";\n";
```

Comment:
Don't do an else, just unconditionally emit the ";\n" (and remove from the above branch).

Reply:
Ok

Reply:
Done